lib/person.rb

```diff
@@ -1,40 +1,46 @@
 class Person
+	attr_accessor :age, :name, :city, :gender
+
 	def initialize(args)
-		@gender = args[:gender]
-		@age    = args[:age]
-		@name   = args[:name]
+		@name   	= args[:name]
+		@city   	= args[:city]
+		@age   		= args[:age]
+		@gender   = args[:gender]
 	end
 
-	def male?
-		@gender == "M"
-	end	
+	def female?
+		gender == "F"
+	end
 
-	def age?
-		milestone = 75
-		age == 75
+	def old?
+		age >= 75
 	end	
 
-def age?
-		milestone = 75
-		age == 32
-	end	
+	def young?
+		age <= 8
+	end
 
+	def local?
+		city = "Atlanta"
+	end
 
-end
+	def grandfather?
+		age >= 70 && gender == "M"
+	end
 
+	def grandmother?
+		age >= 70 && gender == "F"
+	end
 
-		# if age 	+ user[:age]
-		# 	age_delta = (milestone - age).abs
-		# if age > milestone
-		# 	"You turned #{milestone} #{age_delta} years ago!"
-		# elsif age < milestone
-		# 	 "You'll turn #{milestone} in just #{age_delta} years!"		
-		# else 
-		# 	 "Wow, you're #{milestone}!"
-		# end
+	def great_granny?
```

Comment:
No end here. Be adamant about tabbing or you'll set yourself up for confusion and syntax errors. Right now this a method inside another method which is a big no-no.